dist/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon
+  id="webinterface.elm"
```

Comment:
We want to use the old id from chorus 2 here probably, but no hard dependency for merging this

Reply:
Old id is `webinterface.default`. Was running into problems locally since I had chorus2 installed simultaneously but I agree that we can change this. Not sure if we should change it yet since it might break some users experience. Maybe keep `webinterface.elm` for testing until we have something more stable to make default?